src/main.js

```diff
@@ -1 +1,5 @@
-// Write your code here
\ No newline at end of file
+// Write your code here
+console.log(5 + 7); 
+console.log(5 * 7 * 4);       
+console.log("Hello Lizi");    
+console.log("Hello" + " " + "World");
```

Comment:
მხოლოდ ასო ბგერების გამოყენება შეგიძლია. `" "` -ი არაა ასო ბგერა
```suggestion
console.log("Hello" + "World");
```